diff --git a/php74.spec b/php74.spec
index 6a109b2..90e38c6 100644
--- a/php74.spec
+++ b/php74.spec
@@ -1045,7 +1045,10 @@ build --libdir=%{_libdir}/php \
 %if %{with_libgd}
       --with-external-gd \
 %else
+      --with-freetype \
       --with-webp \
+      --with-jpeg \
+      --with-xpm \
 %endif
       --with-gmp=shared \
       --enable-calendar=shared \
@@ -1687,6 +1690,9 @@ exit 0
 
 
 %changelog
+* Wed Apr 29 2020 David Alger <davidmalger@gmail.com> - 7.4.5-1
+- Reintroduce freetype, jpeg and xpm support in GD
+
 * Sun Apr 26 2020 David Alger <davidmalger@gmail.com> - 7.4.5-1
 - Latest upstream
